sbtc-core/src/signer/coordinator/mod.rs

```diff
@@ -25,8 +27,41 @@ pub struct PublicKeys {
 /// TODO: Define the Message types for Tx Signning Round
 /// https://github.com/stacks-network/sbtc/issues/43
 
+/// An sBTC transaction needing to be processed by the coordinator
+/// TODO: replace with the core library's SBTCTransaction
+/// This could be a BTC transaction or a STX transaction
+/// depending on https://github.com/Trust-Machines/stacks-sbtc/pull/595
+pub enum SBTCTransaction {
+    /// A commit Bitcoin transaction
+    Commit(TaprootSpendInfo, BitcoinTransaction),
+    /// A withdrawal Stacks transaction
+    Withdawal(StacksTransaction),
+}
+
+/// Revealer trait for revealing BTC commit transactions
+pub trait Revealer {
```

Comment:
Should the trait be called "Reveal" to be consistent with the "Sign" and "Coordinate" traits?